### PR TITLE
Release 10.0.0 + modulesync

### LINKED ISSUE #817: Version release
Do you think a new version release would be possible soon?
We are interested particularly in https://github.com/voxpupuli/puppet-rabbitmq/pull/814 since the module is currently not working with RabbitMQ version 3.8.0.

## Affected Puppet, Ruby, OS and module versions/distributions

- Puppet: 4.10.12
- Ruby: -
- Distribution: CentOS 7
- Module version: 9.1.0

## How to reproduce (e.g Puppet code you use)

Include module

## What are you seeing

```Failed to apply catalog: undefined method `scan' for nil:NilClass```

## What behaviour did you expect instead

No error

### PR DESCRIPTION
**Pull request by wyardley**
## Pull Request (PR) description
BREAKING CHANGE: Drop Ubuntu 14.04 support (#818)
- Extend version regex for RabbitMQ 3.8 (#814)
- Clean up acceptance spec helper (#815)

#### This Pull Request (PR) fixes the following issues
Fixes #817

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -158,7 +175,6 @@ These should not affect the functionality of the module.
 **Fixed bugs:**
 
 - rabbitmq\_user\_permissions timing out [\#689](https://github.com/voxpupuli/puppet-rabbitmq/issues/689)
-- Set default LC\_ALL =\> en\_US.UTF-8 \(\#671, \#689\) [\#694](https://github.com/voxpupuli/puppet-rabbitmq/pull/694) ([wyardley](https://github.com/wyardley))
```
Comment (ekohl) [inline CHANGELOG.md]: That this moves around could be a caching issue. Can you do a modulesync and use our fork to generate it?

Comment (bastelfreak) [reply CHANGELOG.md]: a modulesync from the master branch of modulesync_config should fix it

**Comment by ekohl** [comment]
Given this is breaking, should it also merge https://github.com/voxpupuli/puppet-rabbitmq/pull/796 to avoid doing too many breaking changes? Perhaps there are more.

File: CHANGELOG.md (hunk 1 above)
Comment (wyardley) [reply CHANGELOG.md]: Doing it, one sec

**Comment by wyardley** [comment]
> Given this is breaking, should it also merge #796 to avoid doing too many breaking changes? 
> Perhaps there are more.

I'm not sure #796 is breaking, but I also don't think it's the right behavior still.

File: CHANGELOG.md (hunk 1 above)
Comment (wyardley) [reply CHANGELOG.md]: Is this right now?

**Comment by wyardley** [comment]
Since modulesync isn't in its own PR, I shouldn't add the modulesync label, right?

**Comment by wyardley** [comment]
@ekohl @bastelfreak I never got feedback on #785, but I think that one would be backwards-incompatible... from a security standpoint, this one is probably a good one to release, even though it changes behavior... however, because of the change of order, I think it could cause a reload of configs?

**Comment by bastelfreak** [comment]
I approved #785. Please also run `bundle exec rake reference` to ensure that the REFERENCE.md is up2date.

**Comment by wyardley** [comment]
Thanks @bastelfreak!

**Comment by wyardley** [comment]
@ekohl @bastelfreak: Updated changelog. Reference seems up-to-date.

**Pull request merged**